doc/whats_new.rst

```diff
@@ -1,13 +1,31 @@
 0.3.1
 =====
 
+Changelog
+---------
+
+    - The function sym_to_vec is is deprecated and will be removed in future
```

Comment:
is is repeated twice. Can you also add release version we want to remove **0.4** ?